h2o-automl/src/main/java/hex/schemas/TargetEncoderV3.java

```diff
@@ -0,0 +1,36 @@
+package hex.schemas;
+
+import ai.h2o.automl.targetencoding.TargetEncoderBuilder;
+import ai.h2o.automl.targetencoding.TargetEncoderModel;
+import water.api.API;
+import water.api.schemas3.ModelParametersSchemaV3;
+
+import java.util.List;
+
+public class TargetEncoderV3 extends ModelBuilderSchema<TargetEncoderBuilder, TargetEncoderV3, TargetEncoderV3.TargetEncoderParametersV3> {
+  public static class TargetEncoderParametersV3 extends ModelParametersSchemaV3<TargetEncoderModel.TargetEncoderParameters, TargetEncoderParametersV3> {
+    
+    @API(help = "Is blending used ? True by default.", mapsTo = "_withBlending")
+    public boolean blending;
+    
+    @API(help = "Columnds to encode.", mapsTo = "_columnNamesToEncode")
+    public String[] encoded_columns;
```

Comment:
ColSpecifierV3[]?

Reply:
`08-14 11:08:16.759 192.168.0.103:54321   20866  #56930-17 FATAL: Don't know how to cast an array of: class water.api.schemas3.FrameV3$ColSpecifierV3 to an array of: class java.lang.String`

Even classic Model.Params do not use array of ColSpecifierV3. I'll try to add the functionality and if it will take too long, use string as the rest of the Model.Parameters instead.

Reply:
I see, VecSpecifiers have to be used. Ok, the code can be adapted without problems.

Reply:
Done.